rna/rna-collect-rna-seq-metrics/README.md

```diff
@@ -0,0 +1,28 @@
+# RNACollectRnaSeqMetrics with Picard
+
+This workflow runs `picard CollectRnaSeqMetrics` on a BAM file
```

Comment:
This workflow runs [picard CollectRnaSeqMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_analysis_CollectRnaSeqMetrics.php) on a BAM file.